presto-docs/src/main/sphinx/sql/select.rst

```diff
@@ -712,6 +712,29 @@ Using multiple columns::
      [7, 8, 9] | 9 | 3
     (5 rows)
 
+FOR ``MAP`` columns::
```

Comment:
it: What about :
```
``UNNEST`` also works for ``MAP`` columns::
```

Reply:
I don't think we need to use `also works for`. 

I will recommend changing this to be in sync with the rest of the document.

Changing - `Using a single column::` to `Using a single array column::`

And then adding this example after the array example with the title `Using a single map column::`